deploy/install-rbac.sh

```diff
@@ -33,7 +33,7 @@ fi
 $DIR/install-bootstrap-creds.sh
 
 # create the Operator service accounts
-expenv -f $DIR/service-accounts.yaml | $PGO_CMD --namespace=$PGO_OPERATOR_NAMESPACE create -f -
+expenv -f $DIR/service-accounts.yaml | $PGO_CMD create -f -
```

Comment:
I thought this change was already getting to be out of scope of the original goal but i can work on this change if you think this would be a good place for it.

Reply:
>Also, does --namespace take precedence over the environmental variable being set in the file? If so, we could just leave --namespace kept in.

Seems like this isn't possible. They will have to be the same namespace
```
[vagrant@jmckulk-centos-cloud ~]$ kubectl create -f sa.yaml -n max
error: the namespace from the provided object "daisy" does not match the namespace "max". You must pass '--namespace=daisy' to perform this operation.
```